stdlib/pdb.pyi

```diff
@@ -59,7 +59,6 @@ class Pdb(Bdb, Cmd):
         def execRcLines(self) -> None: ...
 
     def bp_commands(self, frame: FrameType) -> bool: ...
-    def interaction(self, frame: FrameType | None, traceback: TracebackType | None) -> None: ...
```

Comment:
Nit: I prefer multiple version_info branches and keeping the methods in implementation order rather than bunching them together into one branch. The same is also true for new methods.

Reply:
Sounds good, let me go ahead and fix things up.

Reply:
https://github.com/python/typeshed/pull/12267/commits/3f201b91b62cc7f3a68f9e564737b9f717d2075c